play_tester.py

```diff
@@ -14,11 +14,14 @@
 pp.pprint(FANTASY_1)
 pp.pprint(FANTASY_2)
 
-# print("\n-----Wave 02 user_data-----")
-# pp.pprint(clean_wave_2_data())
+#TODO: put in own tests to run functions to see input
+# print(create_movie("nacho", "comedy", "9.0"))
 
-#print("\n-----Wave 03 user_data-----")
-#pp.pprint(clean_wave_3_data())
+print("\n-----Wave 02 user_data-----")
+pp.pprint(clean_wave_2_data())
```

Comment:
Would advise to omit or comment out print statements in your final submission! In large projects, the console is great for debugging, but when it's used in production or by other developers who might not be familiar with the code, an excess of printing can clutter and confuse them as they look at the console raising questions like “What does this mean, why is it printed, where is this coming from?”